llvm/lib/Target/AMDGPU/AMDGPUCallLowering.cpp

```diff
@@ -520,7 +522,6 @@ bool AMDGPUCallLowering::lowerFormalArgumentsKernel(
 
   // TODO: Align down to dword alignment and extract bits for extending loads.
   for (auto &Arg : F.args()) {
-    // TODO: Add support for kernarg preload.
     if (Arg.hasAttribute("amdgpu-hidden-argument")) {
```

Comment:
I guess support for hidden args can be added separately?

Reply:
should be fixed